app/forms/provider_interface/rejections_wizard.rb

```diff
@@ -26,6 +26,17 @@ def sanitize_attrs(attrs)
       reset_deselected_attrs(attrs)
     end
 
+    def selectable_reasons(application_choice)
```

Comment:
Feels weird passing in an application choice as an argument here

Reply:
Yes, I agree. I really wish to pass somewhere but I definitely didn't wanna change significantly the redis wizards... so I took this route.